test-it/karma/publicapi/WriteJson.Test.js

```diff
@@ -82,10 +83,9 @@ describe('The Vireo EggShell writeJSON api can write', function () {
         });
 
         it('Int64', function () {
```

Comment:
We should add some tests for writing 64 bit numbers as both JSON numbers and JSON strings.